Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -13,7 +12,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// Add a NearInteractionTouchable to your scene and configure a touchable surface
     /// in order to get PointerDown and PointerUp events whenever a PokePointer touches this surface.
     /// </summary>
-    public class NearInteractionTouchable : ColliderNearInteractionTouchable
+    public class NearInteractionTouchable : NearInteractionTouchableSurface
```

Comment:
@wiwei Adam and I were lamenting that NearInteractionTouchable needs to inherit NearInteractionTouchableSurface.

My ideal would be:

NearInteractionTouchableSurface : BaseNearInteractionTouchable
NearInteractionTouchableVolume : BaseNearInteractionTouchable
NearInteractionTouchableUnityUI : BaseNearInteractionTouchable

I'd like to create a plan to get us there. Not needed for this PR since one of MRTK members can do this. 

Something like this:
- Mark NearInteractionTouchable as Deprecated 
- Provide migration guidance "if this is something that should be touched only from the front, use surface, otherwise use volume"?
- In one release after, delete NearInteractionTouchable

I can create an issue to follow up on this

Reply:
@julenka , I thought your ideal was what I've now done (other than the fact that "NearInteractionTouchable" should ideally be called something like "NearInteractionTouchableCollider" or something like that.  For reference, here's the whiteboard drawing we came up with of what I had done in an earlier iteration of this PR vs what is now done.

![mrtk_nearinteraction_class_hierarchy (2)](https://user-images.githubusercontent.com/41760870/64276615-43bcb880-cefd-11e9-8fca-b438fc91ed26.jpg)

Reply:
Oh, sorry. Let me update my ideal. 

NearInteractionTouchableSurface : BaseNearInteractionTouchable
NearInteractionTouchableVolume : BaseNearInteractionTouchable
NearInteractionTouchableUnityUI : NearInteractionTouchableSurface 

And then NearInteractionTouchable no longer exists, instead NearInteractionTouchable would be renamed to NearInteractionTouchableSurface.

I was trying to say that I think the name "NearInteractionTouchable" is misleading, since it's really a touchable surface. It threw me a bit that NearInteractionTouchable : NearInteractionTouchableSurface but then I remembered that was the plan.